nixos/tests/beanstalkd.nix

```diff
@@ -0,0 +1,45 @@
+import ./make-test.nix ({ pkgs, lib, ... }:
+
+let
+  produce = pkgs.writeScript "produce.py" ''
+    #! ${pkgs.python2}/bin/python
```

Comment:
beanstalk should work with python3: https://github.com/bosondata/beanstalkc/pull/1

Reply:
you also might want to use a nix-shell python shebang on top of each python file instead of assembling python paths manually:

```
#! /usr/bin/env nix-shell
#! nix-shell -i python3 -p python3Packages.beanstalkc
```

Reply:
The beanstalkc page said python 2 only so thanks for pointing that out. I'll update beanstalkc in nixpkgs to support python 3.

I had an issue with using nix-shell... I'll try again and report back.

Reply:
You can also get rid of nix-shell entirely, by using this shebang:
```
#!${pkgs.python3.withPackages (p: [p.beanstalkc])}/bin/python
```

Reply:
Awesome that worked! Will replace with python3 when https://github.com/NixOS/nixpkgs/pull/56117 is merged.